src/Framework/Framework/Hosting/ErrorPages/ErrorPageTemplate.cs

```diff
@@ -37,12 +37,16 @@ public class ErrorPageTemplate : IErrorWriter
         public string Summary { get; }
         public IErrorSectionFormatter[] Formatters { get; }
 
-        public void WriteText(string str)
+        public void WriteText(string? str)
```

Comment:
This seems like an unrelated change. Why these changes are here?

Reply:
This only fixes the warnings that we now check in CI